vespa-athenz/src/main/java/com/yahoo/vespa/athenz/tls/AthenzX509CertificateUtils.java

```diff
@@ -68,12 +68,16 @@ public static Optional<String> getInstanceId(X509Certificate cert) {
     /** @return Athenz unique instance id from the Subject Alternative Name extension */
     public static Optional<String> getInstanceId(List<SubjectAlternativeName> sans) {
         // Prefer instance id from SAN URI over the legacy DNS entry
-        return getAthenzUniqueInstanceIdFromSanUri(sans)
+        return getLastSegmentFromSanUri(sans, "athenz://instanceid/")
                 .or(() -> getAthenzUniqueInstanceIdFromSanDns(sans));
     }
 
-    private static Optional<String> getAthenzUniqueInstanceIdFromSanUri(List<SubjectAlternativeName> sans) {
-        String uriPrefix = "athenz://instanceid/";
+    /** @return Athenz unique instance name from the Subject Alternative Name extension */
+    public static Optional<String> getInstanceName(List<SubjectAlternativeName> sans) {
+        return getLastSegmentFromSanUri(sans, "athenz://instancename/");
```

Comment:
The returned value has the format project/instancename - so return a record with those fields?

Reply:
The returned value is only the instance name, since it returns the last path segment